system-parachains/asset-hubs/asset-hub-kusama/src/weights/xcm/mod.rs

```diff
@@ -61,16 +61,8 @@ impl<Call> XcmWeightInfo<Call> for AssetHubKusamaXcmWeight<Call> {
 	fn withdraw_asset(assets: &MultiAssets) -> Weight {
 		assets.weigh_multi_assets(XcmFungibleWeight::<Runtime>::withdraw_asset())
 	}
-	// Currently there is no trusted reserve (`IsReserve = ()`),
```

Comment:
@svyatonik 
as a part of this backport, please, search and remove these two comments:
https://github.com/svyatonik/runtimes/blob/main/relay/polkadot/src/weights/xcm/mod.rs#L94C3-L94C92
```
// Polkadot doesn't support ReserveAssetDeposited, so this benchmark has a default weight
```
https://github.com/svyatonik/runtimes/blob/main/relay/kusama/src/weights/xcm/mod.rs#L94
```
// Kusama doesn't support ReserveAssetDeposited, so this benchmark has a default weight
```